app/chime/ChimeSdkWrapper.ts

```diff
@@ -114,7 +114,7 @@ export default class ChimeSdkWrapper implements DeviceChangeObserver {
   lookupClosestChimeRegion = async (): Promise<RegionType> => {
     let region: string;
     try {
-      const response = await fetch(`https://l.chime.aws`, {
+      const response = await fetch(`https://nearest-media-region.l.chime.aws`, {
```

Comment:
Make this change only but revert other lines back to the previous commit. The classroom demo script is automatically updating all app names.